content/en/graphing/_index.md

```diff
@@ -9,128 +9,27 @@ aliases:
 description: Visualize your data to gain insight
 further_reading:
 - link: "https://learn.datadoghq.com/course/view.php?id=8"
-  tag: "Learning Center"
-  text: "Building Better Dashboards"
+  tag: “Learning Center”
+  text: “Building Better Dashboards”
 ---
 
-Graphs are the window to your monitored systems. You see graphs throughout Datadog, in email notifications, Slack, and other chat clients. Graphs are at the center of monitoring and observability, so it is essential to understand how to define graphs.
+Graphs are the window to your monitored systems. You see graphs throughout Datadog, in email notifications, Slack, and other chat clients. You can define graphs for monitoring your infrastructure using metrics, monitors, dashboards, and notebooks.
```

Comment:
```suggestion
Graphs are the window to your monitored systems. You see graphs throughout Datadog, in email notifications, and Slack and other chat clients. You can define graphs for monitoring your infrastructure using metrics, monitors, dashboards, and notebooks.
```

just changing some punctuation so it doesn't seem like email notifications are a chat client